content/2017.04.getting-started-with-elm.rst

```diff
@@ -0,0 +1,433 @@
+Getting started with Elm
+########################
+
+:lang: en
+:date: 2017-04-13
+:url: getting-started-with-elm
+:summary: What is Elm? how does it work? How can I try it really fast?
+
+Introduction
+============
+
+The past two weeks we have been traveling and meeting a bunch of
+developers.
+
+It was a chance for us to tell them about elm through talks at
+conference and to train a little and short one hour workshop.
+
+You can find the slides of the two talks here:
+
+- `a short introduction to Elm by Niko <http://slides.com/n1k0/elm>`_
+- `Talking JSON with Elm by Natim <http://natim.github.io/django-slides/elm-json/>`_
+
+However, this article is a summary of the take aways of this workshop
+for thoose who would like to know a bit more about Elm.
+
+
+What is Elm and why should I be concerned?
+==========================================
+
+Elm is a toolkit that is focused on making sure your app state and the
+DOM (html nodes) that reflects it are always in sync and that updates
+on your app state are reflected on what the user is seeing.
+
+If you have been doing some frontend during the last couple of years,
+you might have seen that the JavaScript ecosystem is a bit wild.
+
+Keeping up-to-date with the growing number of competing frameworks can
+take a lot of energy, not to mention trying your app on all the
+available browser version to make sure it works well everywhere.
+
+Training a team to build an app with React.js, Angular2.js, Vue.js or
+Ember.js as well as looking for the right boilerplate to get started
+with your project are good examples of how hard the learning curve can
+be.
```

Comment:
This paragraph might also be joined with the previous one. I like this paragraph but something about the wording is not quite right. I think it's because the first thing you mentioned isn't really an example of the learning curve, but rather examples of the frustration that the constant reinvention of the ecosystem causes. I'm not sure about the second thing, but that doesn't seem like part of the learning curve either.